test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1819,7 +1819,7 @@ def verify_manifest(self, expected_manifest):
             elif key == 'metadata':
                 metadata = manifest['metadata']
                 self.verify_metadata(
-                    metadata, 'https://schemas.getdbt.com/dbt/manifest/v4.json')
+                    metadata, 'https://schemas.getdbt.com/dbt/manifest/v5.json')
```

Comment:
FYI until we migrate `029_docs_generate_tests` to use the new framework, move it into the "adapter zone," and inherit/use it within our adapter plugins, this will cause test failures in our plugin repos:
- https://github.com/dbt-labs/dbt-redshift/blob/main/tests/integration/docs_generate_tests/test_docs_generate.py
- https://github.com/dbt-labs/dbt-snowflake/blob/main/tests/integration/docs_generate_tests/test_docs_generate.py
- https://github.com/dbt-labs/dbt-bigquery/blob/main/tests/integration/docs_generate_tests/test_docs_generate.py